### PR TITLE
Generate sources jar for ros2_java_examples

### PR DESCRIPTION
**Pull request by gonzodepedro**
Signed-off-by: Gonzalo de Pedro <gonzalo@depedro.com.ar>

### CODE REVIEW

**Comment by gonzodepedro** [comment]
Currently, this will create a sources jar and place it under '/workspace1/ros2_java_ws/build/rcljava_examples/build/libs'.

File: rcljava_examples/build.gradle [hunk 1]
```diff
@@ -17,6 +17,7 @@ buildscript {
   dependencies {
     classpath 'gradle.plugin.org.ros2.tools.gradle:ament:0.7.0'
   }
+
```
Comment (jacobperron) [inline rcljava_examples/build.gradle]: nitpick: undo this unrelated whitespace change

File: rcljava_examples/build.gradle [hunk 2]
```diff
@@ -42,3 +43,12 @@ ament {
         ]
     }
 }
+
+task sourcesJar(type: Jar, dependsOn: classes) {
+    classifier = 'sources'
+    from sourceSets.main.allSource
```
Comment (jacobperron) [inline rcljava_examples/build.gradle:50]: It would be nice to install the artifact to the ament install space. After some playing around, I think this will install the source jar alongside the class jar:
```suggestion
    destinationDir = new java.io.File(ament.installSpace + "/share/" + ament.packageManifestName + "/java")
    from sourceSets.main.allSource
```

In the future, we might want to add an option to the `ament` gradle plugin.

**Review by jacobperron — COMMENTED** [review]
LGTM, one suggestion related to the install location.

**Review by jacobperron — APPROVED** [review]
